packages/liquid-html-parser/grammar/liquid-html.ohm

```diff
@@ -389,7 +389,17 @@ LiquidStatement <: Liquid {
 }
 
 LiquidDoc <: Helpers {
-  Node := (TextNode)*
+  Node := (LiquidDocNode | TextNode)*
+  LiquidDocNode =
+    | paramNode
+    | fallbackNode
+
+  fallbackNode = "@" anyExceptStar<newline>
+  paramNode = "@param" space* (paramType)? space* (paramName)? space* "-"? paramDescription?
```

Comment:
according to the JSDOc [spec](https://jsdoc.app/tags-param), `paramName` seems to be required.

I'm making it optional to enable formatting formatting to apply even when paramName isn't present.